tests/helpers/test_service.py

```diff
@@ -145,18 +153,23 @@ def test_split_entity_string(self):
             },
         )
         self.hass.block_till_done()
-        assert ["hello.world", "sensor.beer"] == self.calls[-1].data.get("entity_id")
+        assert ["hello.world",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4603109)